controllers/storagecluster/prometheus/localcephrules.yaml

```diff
@@ -278,6 +278,17 @@ spec:
       for: 10m
       labels:
         severity: warning
+    - alert: CephMonLowNumber
+      annotations:
+        description: The number of node failure zones available allow to increase the number of Ceph monitors in order to improve cluster resilience.
```

Comment:
Better to mention the number of zones the cluster have and the count that the user can upgrade to. FYI, the possible values are 3,5 only.
So maybe, we can show something like
`The ceph monitor count in the cluster can be increased to 5 to improves cluster resilience, as the cluster has 5 or more failure zones available` (could be resphrased).
Also, shouldn't we highlight the steps to increase the count?

Reply:
I think we can provide more information about how to increase the number of monitors in the runbook documentation. A long explanation about how to resolve the alert in the alert description maybe is not a good idea.
I will add the runbook file asap this pr will be merged, it is referenced in the `runbook_url` annotation.